invenio_records_resources/services/custom_fields/__init__.py

```diff
@@ -8,15 +8,12 @@
 """Custom Fields for InvenioRDM."""
 
```

Comment:
discussion, not blocking:

it is now still service level? Or now is records (data) layer? could be a separate one?

Reply:
maybe more of a data layer but let's take it when we tackle https://github.com/inveniosoftware/invenio-records-resources/issues/365 !